src/Resources/config/services.xml

```diff
@@ -19,7 +19,7 @@
         </service>
 
         <service id="translation.loader.zicht_messages" class="Zicht\Bundle\MessagesBundle\Translation\Loader">
-            <tag name="translation.loader" alias="db"/>
+            <tag name="translation.loader" alias="zz"/>
```

Comment:
Is er geen priority attribuut? Dat zie je ook nog wel eens bij tags

Reply:
Nope, heb ik ook naar gezocht. Ik hoop ook dat deze aanpassing blijft werken in alle projecten, ik vind het gek dat een eenvoudig volgordeprobleem een door Symfony voorgeschreven recept in de war kan schoppen.

Reply:
Is het niet beter dan om te kijken of wel zelf een priority attribuut kunnen introduceren? (En tegelijkertijd de kwestie bij Symfony zelf aankaarten?)

Reply:
![image](https://user-images.githubusercontent.com/33859362/90769210-280a1800-e2f0-11ea-9fc2-4a0caa8d683f.png)

Het gaat **eigenlijk** mis omdat de bestanden nu bij elkaar in 1 mapje staan, dat was voorheen niet zo, toen stonden alleen de `.db` bestanden in `app/Resources`

Reply:
Dus het heeft niet te maken met de priority van de `Loader`

Reply:
Het heeft te maken met de volgorde van de files?

Het klopt inderdaad dat `app/Resources/` een soort van _override_ locatie is/was. Dus deze locatie werd als laatste ingeladen.